contracts/lib/time_series/structs.cairo

```diff
@@ -0,0 +1,27 @@
+%lang starknet
+
+using NEVER = felt
+
+struct TickElem:
+    member tick : felt
+    member value : felt
+end
+
+struct Matrix2D:
+    member x_dim : felt
+    member y_dim : felt
+    member m : felt**
+end
+
+struct List:
+    member length : felt
+    member size : felt
+    member arr : felt*
+end
+
+struct PAIRWISE_OPERATION:
+    member ADDITION : NEVER  # DEFAULT
```

Comment:
Why `NEVER`?

Reply:
you never actually use the member, it's just there to indicate the index of the storage in the struct for an enum